### ISSUE TITLE
Is this fork still required?

### ISSUE THREAD

**Issue by spuder** [flutterwireless/ArduinoCodebase#7]
The mainline arduino versions are already up to 1.6 and 1.7 (depending if you like arduino.cc or arduino.org forks). 

Is this modified code still required for flutter?

**Comment by sequoia-hope** [comment]
Yes, we currently have a lot of mods for the SAM3S processor and we haven't yet merged them into the main tree. We've also just brought up USB support, which I will push here soon.

**Comment by jonschoning** [comment]
Is this fork still required?
